src/builtins/.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Please add the following:

```js
"parserOptions": {
  "ecmaVersion": 5
}
```

This will protect against future regressions (see #169, #227) since these aren't currently processed by Babel.

Reply:
Ok, I'll do it :-)

Reply:
Done, and I also replaced one `const` and one `let` by `var` in `src/builtins/index.js`.